src/test/java/org/reactome/release/dataexport/utilities/FTPFileUploaderTestUtils.java

```diff
@@ -140,6 +152,38 @@ public static void mockAllFilesSuccessfullyUploadedExceptOne(
 			.storeFile(eq(fileNotUploaded), any(InputStream.class));
 	}
 
+	public static Properties getITTestPropertiesObject() {
+		final String realConfigPropertiesFileName = "real_config.properties";
+		try {
+			return getTestPropertiesObject(realConfigPropertiesFileName);
+		} catch (IOException | URISyntaxException e) {
+			throw new RuntimeException(
+				"ERROR!  To successfully run integration tests, a configuration file with real values for the " +
+				"EuropePMC and NCBI server configuration values must be provided in this project with the file" +
+				"path src/test/resources/" + realConfigPropertiesFileName + ".  For a sample configuration " +
+				"file, see src/main/resources/sample_config.properties.", e
+			);
+		}
+	}
+
+	private static Properties getMockTestPropertiesObject() throws IOException, URISyntaxException {
```

Comment:
Some of these classes seem to have an issue with overuse of the word "mock" in places where non-mock objects are returned. I think that this method will return a real, valid Properties object, right? If so, I don't think "mock" should be in this method's name.

There are a number of methods like this...

Reply:
Yup, see https://github.com/reactome/release-data-exporter/pull/8#discussion_r610895165

Reply:
Okay, everything is now renamed where "mock" should be "dummy" or "test" instead.